cg/store/crud/delete.py

```diff
@@ -36,3 +36,13 @@ def delete_illumina_flow_cell(self, internal_id: str):
             self.session.commit()
         else:
             raise ValueError(f"Illumina flow cell with internal id {internal_id} not found.")
+
+    def delete_sample_from_cases(self, sample_id: int) -> None:
```

Comment:
```suggestion
    def decouple_sample_from_cases(self, sample_id: int) -> None:
```
For me this is a clearer naming.

Reply:
https://github.com/Clinical-Genomics/cg/pull/3507/commits/4a6f467fd8a5b4e8b7353631114254c89a3ac718